site/pt/beta/tutorials/keras/basic_regression.ipynb

```diff
@@ -488,9 +488,9 @@
         "id": "BuiClDk45eS4"
       },
       "source": [
-        "This normalized data is what we will use to train the model.\n",
+        "O dado normalizado é o que nós usaremos para treinar o modelo.\n",
         "\n",
-        "Caution: The statistics used to normalize the inputs here (mean and standard deviation) need to be applied to any other data that is fed to the model, along with the one-hot encoding that we did earlier.  That includes the test set as well as live data when the model is used in production."
+        "Atenção: A estatística usada para normalizar as entradas aqui (média e desvio padrão) precisa ser aplicada em qualquer outro dado que alimenta o modelo, junto com o codificação one-hot que fizemos mais cedo.  Isso inclui o conjunto de teste, assim como dados que o modelo usará em produção."
```

Comment:
'Caution:', 'Note:', and some other tags are used for styling on the website. They should not be translated.
I can fix this one